client/src/docs-ui/internal-link/internal-link.tsx

```diff
@@ -25,9 +25,9 @@ export class DocsInternalLink {
   @State() isChildActive?: boolean;
 
   @Watch("selectedFilters")
-  componentWillLoad() {
+  async componentWillLoad() {
```

Comment:
Does this affect anything on Stencil side since it is there lifecycle method?

Reply:
Stencil's `componentWillLoad` lifecycle method can be async. It does not affect anything.